cmake/modules/kernel.cmake

```diff
@@ -168,6 +168,11 @@ set(KERNEL_STRIP_NAME ${KERNEL_NAME}.strip)
 set(KERNEL_META_NAME  ${KERNEL_NAME}.meta)
 set(KERNEL_SYMBOLS_NAME    ${KERNEL_NAME}.symbols)
 
+# Enable dynamic library support required by CONFIG_XTENSA for CONFIG_LLEXT support.
+if(CONFIG_XTENSA AND CONFIG_LLEXT)
```

Comment:
64e7d85ef670 added a Kconfig to specify the object type for the llext being built, so it's not tied to the arch anymore. 
Thus, I believe this should now be:

```suggestion
# Enable dynamic library support when required by LLEXT.
if(CONFIG_LLEXT AND CONFIG_LLEXT_TYPE_SHAREDLIB)
```

Reply:
Nice. I amended the branch to include your suggestion. Thanks for the tip!

Reply:
Sorry for the misdirection - in fact it's `CONFIG_LLEXT_TYPE_ELF_SHAREDLIB`, my bad! :wink: